### PR TITLE
RavenDB-13296 Expose task count on Manage Ongoing tasks view

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts [hunk 1]
```diff
@@ -73,6 +75,20 @@ class ongoingTasks extends viewModelBase {
     private initObservables() {
         this.myNodeTag(this.clusterManager.localNodeTag());
         this.serverWideBackupUrl = appUrl.forServerWideBackupList();
+        
+        this.taskNameToCount = ko.pureComputed<dictionary<number>>(() => {
+            var tasksNameToCount: { [id: string] : number; } = {};
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: never use var in typescript. Const or let.

File: src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts [hunk 2]
```diff
@@ -73,6 +75,20 @@ class ongoingTasks extends viewModelBase {
...
+          
+            tasksNameToCount["External Replication"] = this.replicationTasks().length;
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: and return this as object w/o assigning props line by line. I know you spent last week in java, but hopefully this is TS and we have nice syntax for doing this. ;-)

Comment (ml054) [reply src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: so it goes like this:

return {
   "External replication" : this.replicationTasks().length,
....
}

File: src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: right - can't believe I overlooked that  :)

File: src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: done - thanks

File: src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts [hunk 3]
```diff
@@ -369,7 +374,8 @@ class ongoingTasks extends viewModelBase {
                     default:
                         return taskType;
                 }
-            }));
+            })
+            .sort((a, b) => { return this.tasksOrderForUI[a] - this.tasksOrderForUI[b] }));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/tasks/ongoingTasks.ts]: does this sort stable? https://www.geeksforgeeks.org/stability-in-sorting-algorithms/

**Review by ml054 — APPROVED** [review]

**Pull request merged**